src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -294,26 +304,32 @@ class DemuxFastqs
  @arg(doc="Platform model to insert into the group header (ex. miseq, hiseq2500, hiseqX)") val platformModel: Option[String] = None,
  @arg(doc="Comment(s) to include in the merged output file's header.", minElements = 0) val comments: List[String] = Nil,
  @arg(doc="Date the run was produced, to insert into the read group header") val runDate: Option[Iso8601Date] = None,
- @arg(doc="Output gzipped FASTQs (`.fastq.gz`) instead of BAM files") val outputFastqs: Boolean = false,
+ @arg(doc="The type of outputs to produce.") var outputType: OutputType = DemuxFastqs.DefaultOutputType,
+ @deprecated("Use outputType instead.", since="0.5.0")
+ @arg(doc="*** Deprecated: use --output-type instead ***. Output gzipped FASTQs (`.fastq.gz`) instead of BAM files") val outputFastqs: Boolean = false,
```

Comment:
I was considering putting a `mutex` on the two options, so that you can specify them both.  What do you think?

Reply:
Yeah, I think probably the right thing to do until you remove the old option would be:
1. Make the old one `Option[Boolean] = None`
2. Make the new one `Option[OutputType] = None`
3. Add a mutex
4. If both are None, default to Bam, else prefer new option, else old option